src/components/BooksForm.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const booksCategories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+const BooksForm = () => (
+  <div>
+    <form>
+      <label htmlFor="title">
+        Title
+        <input id="title" name="title" type="text" />
+      </label>
+      <label htmlFor="options">
+        Title
+        <select name="options" id="options">
+          {booksCategories.map((category) => (
```

Comment:
- [ ] _[REQUIRED]_ This file should be placed in the `src/containers` directory.
- [ ] _[REQUIRED]_ It is the `BookForm` component not `BooksForm` as you have in your code.